code/modules/antagonists/changeling/powers/mutations.dm

```diff
@@ -558,3 +558,88 @@
+	icon_state = "flesh_mail"
+	item_state = "flesh_maul"
+	lefthand_file = 'icons/mob/inhands/antag/changeling_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/antag/changeling_righthand.dmi'
+	item_flags = NEEDS_PERMIT | ABSTRACT | DROPDEL
+	w_class = WEIGHT_CLASS_HUGE
+	tool_behaviour = TOOL_MINING
+	weapon_stats = list(SWING_SPEED = 2, ENCUMBRANCE = 0, ENCUMBRANCE_TIME = 0, REACH = 1, DAMAGE_LOW = 0, DAMAGE_HIGH = 0)	//Heavy and slow
+	force = 36					//SHATTER BONE
+	throwforce = 0 				//Just to be on the safe side
+	throw_range = 0
+	throw_speed = 0
+	armour_penetration = -20	//Armor will help stop some of the damage
+	wound_bonus = 30			//But your bones will be sad :(
+	hitsound = "swing_hit"
+	attack_verb = list("smashed", "slammed", "crushed", "whacked")
+	sharpness = SHARP_NONE
+	var/can_drop = FALSE
+	var/fake = FALSE
+	resistance_flags = ACID_PROOF
+
+/obj/item/melee/flesh_maul/Initialize(mapload,silent,synthetic)
+	. = ..()
+	ADD_TRAIT(src, TRAIT_NODROP, CHANGELING_TRAIT)
+	if(ismob(loc) && !silent)
+		loc.visible_message(span_warning("A grotesque blade forms around [loc.name]\'s arm!"), span_warning("Our arm twists and mutates, transforming it into a deadly blade."), span_italics("You hear organic matter ripping and tearing!"))
+	if(synthetic)
+		can_drop = TRUE
+
+/obj/item/melee/flesh_maul/afterattack(atom/target, mob/user, proximity)	
+	. = ..()
+	if(!proximity)
+		return
+	if(iscarbon(target))
+		var/mob/living/carbon/C = target
+		C.add_movespeed_modifier("flesh maul", update=TRUE, priority=101, multiplicative_slowdown=1)						//Slows the target because big whack
+		addtimer(CALLBACK(C, /mob.proc/remove_movespeed_modifier, "flesh maul"), 2 SECONDS, TIMER_UNIQUE|TIMER_OVERRIDE)	
+		to_chat(target, span_danger("You are staggered from the blow!"))
+
+	else if(iscyborg(target))
+		var/mob/living/silicon/robot/R = target
+		R.Paralyze(1 SECONDS)						//One second stun on borgs because they get their circuits rattled or something
+
+	else if(istype(target, /obj/structure/table))	//Hate tables
+		var/obj/structure/table/T = target
+		T.deconstruct(FALSE)
+		playsound(src, 'sound/effects/bang.ogg', 50, 1)
+		return
+
+	else if(isstructure(target) || ismachinery(target))	
+		var/obj/structure/S = target			//Works for machinery because they have the same variable that does the same thing
+		var/structure_damage = S.max_integrity		
+		var/make_sound = TRUE
+		if(istype(target, /obj/structure/window) || istype(target, /obj/structure/grille))
+			structure_damage *= 2 				// Because of melee armor
+			make_sound = FALSE
+		if(ismachinery(target) || istype(target, /obj/structure/door_assembly))
+			structure_damage *= 0.5
+		if(istype(target, /obj/machinery/door/airlock))
+			structure_damage = 14				//Won't get bonus damage vs reinforced airlocks but still does the base damage
+		if(target)
+			S.take_damage(structure_damage, BRUTE, "melee", 0)
+		if(make_sound)
+			playsound(src, 'sound/effects/bang.ogg', 50, 1)
```

Comment:
```suggestion
	else if(isstructure(target) || ismachinery(target))	
		var/obj/structure/S = target			//Works for machinery because they have the same variable that does the same thing
		var/structure_damage = S.max_integrity		
		var/make_sound = TRUE
		if(istype(target, /obj/structure/window) || istype(target, /obj/structure/grille))
			structure_damage *= 2 				// Because of melee armor
			make_sound = FALSE
		if(ismachinery(target) || istype(target, /obj/structure/door_assembly))
			structure_damage *= 0.5
		if(istype(target, /obj/machinery/door/airlock))
			structure_damage = 14				//Won't get bonus damage vs reinforced airlocks but still does the base damage
		if(istype(target, /obj/structure/table))	//Hate tables
			var/obj/structure/table/T = target
			T.deconstruct(FALSE)
		if(!isnull(target))
			S.take_damage(structure_damage, BRUTE, "melee", 0)
		if(make_sound)
			playsound(src, 'sound/effects/bang.ogg', 50, 1)
```